### PR TITLE
Add Annual General Meeting

### PR DESCRIPTION
**Pull request by erinosher**
Do not merge. /amend and /nominate links are not live yet

### CODE REVIEW

**Comment by erinosher** [comment]
hacksoc.net/amend and hacksoc.net/nominate are both live

@DanielCordell  can I get a review?

File: src/_events/eventsManifest.json [hunk 1]
```diff
@@ -297,6 +297,17 @@
       "description": "mlh-localhost-learn-c-sharp-with-unity-feb-2019/description.md",
       "banner": "mlh-localhost-learn-c-sharp-with-unity-feb-2019/banner.jpg"
     },
+    {
```
Comment (DanielCordell) [inline src/_events/eventsManifest.json:300]: You put this in a weird place in the list but that's really just stylistic.

**Review by DanielCordell — DISMISSED** [review]
Apart from that one nitpick, lgtm. Maybe also have @AakashDadhich have a look at wording and stuff?

File: src/_events/eventsManifest.json (hunk 1 above)
Comment (erinosher) [reply src/_events/eventsManifest.json:300]: It's not a huge deal. We should probably move away from a single manifest file, to one that generates in the build later.

**Review by DanielCordell — APPROVED** [review]
lgtm

**Pull request merged**